src/enhanced-switch.jsx

```diff
@@ -187,11 +183,6 @@ const EnhancedSwitch = React.createClass({
     window.removeEventListener('resize', this._handleResize);
```

Comment:
Thanks! Didn't realize this component was using the mixin and doing it manually!